guidelines/terms/20/synchronized-media.html

```diff
@@ -5,4 +5,5 @@
       interactive components, unless the media is a <a>media alternative for text</a> that is clearly labeled as such
    </p>
    
+   <p class="note">"Synchronized" refers to all audio and video content presented in the same multimedia experience (whether combined in a single file or played in the same time base). Images and sounds that are out of synch with each other (intentionally or unintentionally) are included by the definition. Asynchronous sound, for example, is still being experienced in the context of time-based, synchronized media.</p>
```

Comment:
```suggestion
   <p class="note">“"Synchronized" refers to all audio and video content presented in the same multimedia experience (whether combined in a single file or played in the same time base). As well as multimedia that is obviously syncronized (for example, seeing people speaking), a presentation with a background track would also considered sycronized. If the multimedia is not "presenting information", it would not require audio description.</p>
```